Gems/OpenXRVk/Code/Include/OpenXRVk/Platform/Linux/OpenXRVk_Linux.h

```diff
@@ -12,10 +12,7 @@
 #include <AzCore/std/algorithm.h>
 #include <limits.h>
 
-#if defined(PAL_TRAIT_LINUX_WINDOW_MANAGER_XCB)
-    #include <xcb/xcb.h> // needed for glad vulkan
-#endif
-#include <glad/vulkan.h>
```

Comment:
Note: I'm not sure if on linux `xcb.h` is needed anymore when using `#include <vulkan/vulkan.h>`, I'll check linux separately and do a follow up PR if necessary.

Reply:
I was able to compile on linux, so `xcb.h` was not needed in this case.